### DISCUSSION TITLE
Translation to Brazilian Portuguese

### DISCUSSION THREAD

**Discussion by brunotech** [microsoft/generative-ai-for-beginners#198] [Ideas]
I've forked the project and started translating it into PT-BR, both the text and video dubbing.

https://github.com/brunotech/generative-ai-for-beginners/tree/main/01-introduction-to-genai/translations/pt-br

**Comment by TimothyMichaelGould** [comment]
This is really cool! Love to watch videos in Portuguese! I can understand almost everything said in so many videos.

**Reply by sailxjx** [reply]
Try this https://codekidz.ai/lesson-intro/generative-a-362093

Though the course content is in English, but you can chat in PT, and listen to the speech in PT
